cxx-qt-gen/src/extract.rs

```diff
@@ -417,23 +417,13 @@ fn extract_type_ident(
     })
 }
 
-/// The result of extracting invokables from an impl block.
-/// Only intended for internal use for now.
-struct ExtractedInvokables {
-    /// Impl methods that will also be exposed as invokables to Qt
-    invokables: Vec<Invokable>,
-    /// Impl methods that will only be visible on the Rust side
-    normal_methods: Vec<ImplItemMethod>,
```

Comment:
As mentioned, we still need these `normal_methods`.

Reply:
Done